js/about/passwords.js

```diff
@@ -64,6 +64,7 @@ class SiteItem extends React.Component {
 
   onDelete () {
     aboutActions.deletePassword(this.props.site.toJS())
+    ipc.send(messages.UPDATE_PASSWORD_SITE_DETAILS)
```

Comment:
nothing additional is needed here. I would suggest creating a passwordsReducer and handling `aboutAction.deletePassword` in that